frontend/src/ui/time.tsx

```diff
@@ -30,23 +36,39 @@ export const RelativeDate: React.FC<RelativeDateProps> = ({ date }) => {
     const MONTH = 30.5 * DAY;
     const YEAR = 365.25 * DAY;
 
+    const liveDate = ((time: number, unit: Intl.RelativeTimeFormatUnit) => {
+        const parts = intl.formatToParts(time, unit);
+        return String(parts[1].value) + parts[2].value;
+    });
+
     const prettyDate = (() => {
-        const intl = new Intl.RelativeTimeFormat(i18n.language);
+        let time: number;
+        let unit: Intl.RelativeTimeFormatUnit;
         if (secsDiff <= 55) {
-            return intl.format(-secsAgo, "second");
+            time = secsAgo;
+            unit = "second";
         } else if (secsDiff <= 55 * MINUTE) {
-            return intl.format(-Math.round(secsAgo / MINUTE), "minute");
+            time = Math.round(secsAgo / MINUTE);
+            unit = "minute";
         } else if (secsDiff <= 23 * HOUR) {
-            return intl.format(-Math.round(secsAgo / HOUR), "hour");
+            time = Math.round(secsAgo / HOUR);
+            unit = "hour";
         } else if (secsDiff <= 6 * DAY) {
-            return intl.format(-Math.round(secsAgo / DAY), "day");
+            time = Math.round(secsAgo / DAY);
+            unit = "day";
         } else if (secsDiff <= 3.5 * WEEK) {
-            return intl.format(-Math.round(secsAgo / WEEK), "week");
+            time = Math.round(secsAgo / WEEK);
+            unit = "week";
         } else if (secsDiff <= 11 * MONTH) {
-            return intl.format(-Math.round(secsAgo / MONTH), "month");
+            time = Math.round(secsAgo / MONTH);
+            unit = "month";
         } else {
-            return intl.format(-Math.round(secsAgo / YEAR), "year");
+            time = Math.round(secsAgo / YEAR);
+            unit = "year";
         }
+        return isLive
+            ? t("video.live-since") + " " + liveDate(time, unit)
```

Comment:
Have you considered what happens when the live event is upcoming, i.e. hasn't started yet? If I'm reading the code correctly, that would result in something weird like "live since -3s". I guess we need to special case that too and add another translation (e.g. "live in ...").

Reply:
You are correct. Also, as you already noted in #508, we should definetly be using the starting time instead of creation time, which I somehow overlooked before.